core/src/main/java/com/tapptitude/core/usecase/LoadImageUseCase.kt

```diff
@@ -4,7 +4,6 @@ import com.tapptitude.core.model.Image
 import com.tapptitude.core.repository.ImageRepository
 
 class LoadImageUseCase(private val imageRepository: ImageRepository) {
```

Comment:
I believe we should make the constructor `internal`, so only we can instantiate it.